loadgen/generic_executor.go

```diff
@@ -0,0 +1,136 @@
+package loadgen
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"go.temporal.io/sdk/client"
+	"go.uber.org/zap"
+)
+
+type GenericExecutor struct {
```

Comment:
Not loving this name, "shared iterations" is more descriptive than "generic" and we will likely want different executors with different behaviors.

Reply:
I don't think the common one needs to have a descriptive name. That is not very Go like. When we get to that other one, we can rename.

Reply:
Sure, can postpone AFAIC